var/spack/repos/builtin/packages/axom/package.py

```diff
@@ -51,7 +51,7 @@ class Axom(CMakePackage, CudaPackage):
     homepage = "https://github.com/LLNL/axom"
     git      = "https://github.com/LLNL/axom.git"
 
-    version('master', branch='master', submodules=True)
+    version('main', branch='main', submodules=True)
```

Comment:
This version will sort as being older than all other stable releases, is this what you want?

Reply:
does spack have special support for `master`, could that support also include `main`?

Reply:
`master` branch no longer exists ...

Reply:
We can name the version whatever we want. We currently have support for `develop`, `master`, `head`, and `trunk`. We could add `main`, or we could do:
```suggestion
    version('master', branch='main', submodules=True)
```

Reply:
Spoke to @becker33, added fix from him that added main to the list of blessed branches.